src/scaling_options_manager.ts

```diff
@@ -24,13 +24,14 @@ export default class ScalingManager {
         ctx: Context,
     ): Promise<FullScalingOptionsResponse> {
         ctx.logger.info(
-            `Updating scaling options for groups of type ${request.instanceType} in region ${request.region}, scaling ${request.direction}`,
+            `Updating scaling options for groups environment ${request.environment} of type ${request.instanceType} in region ${request.region}, scaling ${request.direction}`,
```

Comment:
Please add the environment to these log messages as well:

`No groups of type ${request.instanceType} were found to update in region ${request.region}.
`Groups of type ${request.instanceType} from region ${request.region} are now updated with the new scaling options`,

Reply:
done